psi4/driver/procrouting/dft_funcs/dict_hyb_funcs.py

```diff
@@ -0,0 +1,494 @@
+#
+# @END LICENSE
+#
+"""
+List of hybrid functionals
+"""
+
+import copy
+
+funcs = []
+
+
+
+funcs.append({
+    "name": "B5050LYP",
+    "x_functionals": {
+        "LDA_X": {
+            "alpha": 0.08
+        },
+        "GGA_X_B88": {
+            "alpha": 0.42
+        }
+    },
+    "x_hf": {
+        "alpha": 0.50
+    },
+    "c_functionals": {
+        "LDA_C_VWN": {
+            "alpha": 0.19
+        },
+        "GGA_C_LYP": {
+            "alpha": 0.81
+        }
+    },
+    "description": 
+    '    B5050LYP Hyb-GGA Exchange-Correlation Functional\n',
+    "citation": 
+    '    Y. Shao et. al., J. Chem. Phys., 188, 4807-4818, 2003\n',
+})# yapf: disable
+
+funcs.append({
+    "name": "wPBE0",
+    "alias": ["LC-WPBE0"],
+    "x_functionals": {
+        "GGA_X_HJS_PBE": {
+            "omega": 0.3,
+            "alpha": 0.75
+        }
+    },
+    "x_hf": {
+        "alpha": 0.25,
+        "beta": 1.0,
+        "omega": 0.3
+    },
+    "c_functionals": {
+        "GGA_C_PBE": {}
+    },
+    "description":
+    '    PBE0 SR-XC Functional (HJS Model)\n',
+    "citation":
+    '    Henderson et. al., J. Chem. Phys., 128, 194105, 2008\n' + \
+    '    Weintraub, Henderson, and Scuseria, J. Chem. Theory. Comput., 5, 754, 2009\n',
+})# yapf: disable
+
+funcs.append({
+    "name": "wPBE",
+    "alias": ["LC-WPBE", "LCWPBE"],
+    "x_functionals": {
+        "GGA_X_HJS_PBE": {
+            "omega": 0.4
+        }
+    },
+    "x_hf": {
+        "alpha": 0.0,
+        "beta": 1.0,
+        "omega": 0.4
+    },
+    "c_functionals": {
+        "GGA_C_PBE": {}
+    },
+    "description":
+    '    PBE SR-XC Functional (HJS Model)\n',
+    "citation":
+    '    Henderson et. al., J. Chem. Phys., 128, 194105, 2008\n' + \
+    '    Weintraub, Henderson, and Scuseria, J. Chem. Theory. Comput., 5, 754, 2009\n',   
+})# yapf: disable
+
+# funcs.append({
+#     "name": "wB97X-D3",
+#     "alias": ["WB97X-D3ZERO"],
+#     "x_functionals": {
+#         "HYB_GGA_XC_WB97X": {
+#             "omega": 0.25,
+#             "tweaks": [0.804272, 0.698900, 0.508940, -3.744903, 10.060790,
+#                        1.000000, 2.433266, -15.446008, 17.644390, -8.879494,
+#                        1.000000, -4.868902, 21.295726, -36.020866, 19.177018]
+#         }
+#     },
+#     "x_hf": {
+#         "alpha": 0.195728,
+#         "beta": 1.0,
+#         "omega": 0.25
+#     },
+#     "c_functionals": {},
+#     "dispersion": {
+#         "type": "d3zero",
+#         "params": {
+#             's6': 1.0,
+#             's8': 1.000,
+#             'sr6': 1.281,
+#             'sr8': 1.094,
+#             'alpha6': 14.0
+#         }
+#     },
+#     "description":
+#     '    This functional is currently broken - wB97X needs enabled tweaks.\n',
+#     "citation":
+#     '    Y.-S. Lin, G.-D. Li, S.-P. Mao, J.-D. Chai, J. Chem. Theory Comput., 9, 263-272, 2013\n',
+# })# yapf: disable
+
+funcs.append({
+    "name": "HF",
+    "alias": ["SCF"],
```

Comment:
@psi4/editors do we want SCF as a HF alias – I've been trying to push the more specific HF, with SCF being potentially HF or DFT. I don't mind `energy('scf')` in an input, but if this goes here, then `SCF-D3BJ` shows up in the DFT functionals docs table.

Reply:
FWIW, it was a superfunc before this rework.

Reply:
I believe it. More trying to gauge if others feel a crack-down is in order. I'll need to filter `SCF` out of the functionals table.